libpod/container_internal_freebsd.go

```diff
@@ -190,7 +192,7 @@ func openDirectory(path string) (fd int, err error) {
 }
 
 func (c *Container) addNetworkNamespace(g *generate.Generator) error {
-	if c.config.CreateNetNS {
+	if c.config.CreateNetNS && c.state.NetNS != nil {
```

Comment:
Should this error out if `c.state.NetNS == nil` otherwise it just exits cleanly with a `nil` as if nothing happened and operation `g.AddAnnotation` was successful.

Reply:
Good point - I changed it to return an error if this every happens again.